### PR TITLE
Integration with data pipeline api for parameters

### PR DESCRIPTION
**Pull request by vinopm**

### CODE REVIEW

**Comment by vinopm** [comment]
@sam-brett @johnnonweiler this works AFAIK, we may have to change the config.yaml/metadata.yaml file a bit when we try the integrated run today. Will update the PR if we end up changing the configuration, but the rest is ready to be reviewed

**Comment by johnnonweiler** [comment]
@sam-brett - Merging this with the latest develop was hard.  It was so tricky that I gave up trying to merge everything, and removed the new ageDependence feature.  There were _many_ manual changes required to fix merge conflicts.  Please could you check the merge commit?

In particular, please check AlertEventProcessor.java - I changed the behaviour of timeInStatusAndTestQueue() where I was confident the old behaviour was wrong. I suspect that other bits may also have been incorrect - returning 1 day where they should be returning 1 time step.  Also, it is likely that I have made mistakes in here!

I have changed one of the toml files, so the files on this branch no longer correspond to the files which were uploaded to the data pipeline.

The new infectionRates.json parameters should probably come from the data pipeline, but I have not yet made that change.

File: src/main/java/uk/co/ramp/policy/isolation/BoundedDistributionSerializer.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+              }
+            });
+    gen.writeEndObject();
+  }
+
+  public void write(JsonGenerator gen, Entry<String, JsonNode> entry) {
+    try {
+      gen.writeObjectField(entry.getKey(), entry.getValue());
+    } catch (IOException e) {
+      e.printStackTrace();
```
Comment (sam-brett) [inline src/main/java/uk/co/ramp/policy/isolation/BoundedDistributionSerializer.java:62]: This exception should be handled

File: build.gradle [hunk 2]
```diff
@@ -30,7 +31,9 @@ dependencies {
     implementation 'com.fasterxml.jackson.dataformat:jackson-dataformat-csv:2.11.0'
     implementation "org.immutables:value-annotations:2.8.8"
     implementation "org.immutables:gson:2.8.8"
```
Comment (sam-brett) [inline build.gradle:33]: I guess this is tech debt that I can deal with, but GSON can be removed in favour of Jackson now. GSON is only used in tests now.

File: src/main/java/uk/co/ramp/policy/isolation/BoundedDistributionSerializer.java (hunk 1 above)
Comment (sam-brett) [inline src/main/java/uk/co/ramp/policy/isolation/BoundedDistributionSerializer.java:62]: This exception should be handled

File: src/main/java/uk/co/ramp/policy/isolation/BoundedDistributionSerializer.java [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+import org.apache.commons.math3.random.RandomGenerator;
+import uk.co.ramp.distribution.BoundedDistribution;
+import uk.ramp.mapper.DataPipelineMapper;
+
+/**
+ * This class provides a temporary serialization wrapper over the distribution from std api. This
+ * can/will be removed once we migrate ALL 'distribution' related objects write to the data
+ * pipeline.
+ */
+public class BoundedDistributionSerializer extends JsonSerializer<BoundedDistribution> {
```
Comment (sam-brett) [inline src/main/java/uk/co/ramp/policy/isolation/BoundedDistributionSerializer.java:22]: This class isn't tested

**Review by sam-brett — APPROVED** [review]
@johnnonweiler Happy to approve and I'll fix the age dependency and other bits in another PR.

**Pull request merged**